src/widget.rs

```diff
@@ -98,6 +111,45 @@ impl<O: IsA<Widget> + IsA<Object>> WidgetExtManual for O {
         }
     }
 
+    #[cfg(any(feature = "v3_8", feature = "dox"))]
+    fn add_tick_callback<F>(&self, func: F) -> u32
+    where
+        F: FnMut(&Self, &FrameClock) -> Continue + 'static,
+    {
+        unsafe extern "C" fn add_tick_callback_trampoline(
```

Comment:
Both internal function need be generalized by `O: IsA<Widget> + IsA<Object>` (maybe without `Object`)

Reply:
Ah Ok! Sorry about that.

Reply:
Why? All the other signal connectors use `&Self` too